assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/config/DataCiteConfigurationFactoryTest.java

```diff
@@ -44,40 +54,76 @@ void setUp() {
         setupSystemUnderTest();
     }
 
+    @Test
+    void constructorWithExampleConfigAsInputstreamThenContains2KnownCustomers()
+        throws DataCiteMdsConfigValidationFailedException {
+        dataCiteConfigurationFactory = createDataCiteConfigurationFactoryFromInputStream();
+
+        assertThat(dataCiteConfigurationFactory.getNumbersOfConfiguredCustomers(), is(equalTo(3)));
+        var customerConfig1 = dataCiteConfigurationFactory.getConfig(KNOWN_CUSTOMER_ID);
+        assertThat(customerConfig1.getInstitution(), is(equalTo(KNOWN_CUSTOMER_ID)));
+        var customerConfig2 = dataCiteConfigurationFactory.getCredentials(KNOWN_CUSTOMER2_ID);
+        assertThat(customerConfig2.getInstitution(), is(equalTo(KNOWN_CUSTOMER2_ID)));
+        assertThat(customerConfig2.getDataCiteMdsClientPassword(), is(equalTo(KNOWN_CUSTOMER2_PASSWORD)));
+    }
+
     @Test
     void getCredentialsWithValidConfigurationReturnsSecretInstanceType()
         throws DataCiteMdsConfigValidationFailedException {
```

Comment:
it does not throw the exception

Reply:
Good catch, it had to be turn into a RuntimeException on last PR